x/reward/client/cli/query_reward_pool.go

```diff
@@ -14,7 +14,7 @@ import (
 func CmdListRewardPool() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list-reward-pool",
-		Short: "list all RewardPool",
+		Short: "list all RewardPools",
```

Comment:
```suggestion
		Short: "list all reward pools",
```
It was `RewardPool` in the sense of the name of the structure, we can remove the camel case as well